ceno_zkvm/src/tables/ram/ram_impl.rs

```diff
@@ -369,7 +388,23 @@ impl<DVRAM: DynVolatileRamTable + Send + Sync + Clone> DynVolatileRamTableConfig
     ) -> Result<RowMajorMatrix<F>, ZKVMError> {
         assert!(final_mem.len() <= DVRAM::max_len(&self.params));
         assert!(DVRAM::max_len(&self.params).is_power_of_two());
-        let mut final_table = RowMajorMatrix::<F>::new(final_mem.len(), num_witness);
+
+        let params = self.params.clone();
+        // TODO: make this more robust
+        let addr_column = 0;
```

Comment:
we can clean up this TODO via
`let addr_column = self.addr.id as u64;`

Reply:
Ah, indeed :+1: